api/src/Entity/Activity.php

```diff
@@ -3,48 +3,84 @@
 namespace App\Entity;
 
 use ApiPlatform\Core\Annotation\ApiResource;
-use App\Repository\ActivityRepository;
+use Doctrine\Common\Collections\ArrayCollection;
 use Doctrine\ORM\Mapping as ORM;
 
 /**
- * @ORM\Entity(repositoryClass=ActivityRepository::class)
+ * @ORM\Entity()
  */
 #[ApiResource]
-class Activity extends BaseEntity {
+class Activity extends AbstractContentNodeOwner implements BelongsToCampInterface {
     /**
-     * @ORM\Column(type="text")
+     * @ORM\OneToMany(targetEntity="ScheduleEntry", mappedBy="activity", orphanRemoval=true)
+     *
+     * @var ScheduleEntry[]
      */
-    public ?string $title = null;
+    public $scheduleEntries;
```

Comment:
*ToMany Associations sollten wir alle noch mit `Collection` annotieren. `@var ScheduleEntry[]` ist aber gut, und das hilft API Platform im Notfall (falls Doctrine das nicht sowieso schon weiss), herauszufinden welchen Typ die Einträge in dieser Collection haben.
```suggestion
    public Collection $scheduleEntries;
```